assets/js/blocks/classic-template/constants.ts

```diff
@@ -69,4 +75,24 @@ export const TEMPLATES: TemplateDetails = {
 		),
 		placeholder: PLACEHOLDERS.archiveProduct,
 	},
+	cart: {
+		type: TYPES.cart,
+		title: __( 'WooCommerce Cart Block', 'woo-gutenberg-products-block' ),
+		placeholder: 'cart',
+	},
+	checkout: {
+		type: TYPES.checkout,
+		title: __( 'Checkout Block', 'woo-gutenberg-products-block' ),
+		placeholder: 'checkout',
+	},
+	'checkout-header': {
+		type: TYPES.checkoutHeader,
+		title: __( 'Checkout Header', 'woo-gutenberg-products-block' ),
```

Comment:
I find `Checkout Header` to be too vague. Is `Checkout Page Header` more appropriate?

Reply:
I prefer the short form; the template itself in the editor is not called "Checkout Page", its just labelled as "Checkout". What else could it be referring to?